www/js/metrics/footprint/FootprintComparisonCard.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useContext, useMemo } from 'react';
+import { Card, Text } from 'react-native-paper';
+import { metricsStyles } from '../MetricsScreen';
+import BarChart from '../../components/BarChart';
+import { useTranslation } from 'react-i18next';
+import { ChartRecord } from '../../components/Chart';
+import TimelineContext from '../../TimelineContext';
+import { formatIsoNoYear } from '../../util';
+
+const FootprintComparisonCard = ({
+  type,
+  unit,
+  userCumulativeFootprint,
+  groupCumulativeFootprint,
+  title,
+  addFootnote,
+  axisTitle,
+  goals,
+  showUncertainty,
```

Comment:
In the UI, no, provisional calculations are not indicated. This uncertainty is only for unlabeled trips (or trips labeled with a mode where the footprint is unknown)
`e-mission-common` returns metadata for every calculation which has `is_provisional` and more, but the UI is not using it.
Do you have ideas for how to quantify, or otherwise indicate, the uncertainty of provisional data?